easybuild/easyconfigs/d/DualSPHysics/DualSPHysics-5.0.175-GCC-11.2.0-CUDA-11.4.1.eb

```
@@ -0,0 +1,65 @@
+##
+# Authors:   Eduardo Miguel Firvida Donestevez <efirvida@gmail.com>
+#            Robert Mijakovic                  <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'DualSPHysics'
+version = '5.0.175'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://dual.sphysics.org/'
+
+description = """
+DualSPHysics is based on the Smoothed Particle Hydrodynamics model named SPHysics.
+
+The code is developed to study free-surface flow phenomena where Eulerian methods
+can be difficult to apply, such as waves or impact of dam-breaks on off-shore
+structures. DualSPHysics is a set of C++, CUDA and Java codes designed to deal
+with real-life engineering problems.
+"""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/DualSPHysics/DualSPHysics/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s.patch']
+checksums = [
+    'd65bb7b8b4a21582091a534142348e46cff0a288a97ef86643bc2dde15637575',
+    '24ae21134099252a597d80f25cf835dd981d0f085ce33dc801e70f1d7e045829',
+]
+
+srcdir = 'src/source'
+
+builddependencies = [
+    ('CUDA', '11.4.1', '', True),
+    ('CMake', '3.21.1'),
+]
+
+separate_build_dir = True
+
+buildopts = " && chmod +x %(builddir)s/%(name)s-%(version)s/bin/linux/*_linux64"
+
+install_cmd = """mkdir -p %(installdir)s/bin &&
+              cp -r %(builddir)s/%(name)s-%(version)s/bin/linux/* %(installdir)s/bin &&
+              cp -r %(builddir)s/%(name)s-%(version)s/src/lib/linux_gcc %(installdir)s/lib"""
+
+
+sanity_check_paths = {
+    'files': [
+        'bin/GenCase_linux64',
+        'bin/PartVTK_linux64',
+        'bin/IsoSurface_linux64',
+        'bin/MeasureTool_linux64',
+        'bin/GenCase_MkWord_linux64',
+        'bin/DualSPHysics4.0_LiquidGas_linux64',
+        'bin/DualSPHysics4.0_LiquidGasCPU_linux64',
+        'bin/DualSPHysics5.0_linux64',
+        'bin/DualSPHysics5.0CPU_linux64',
+        'bin/DualSPHysics5.0_NNewtonian_linux64',
+        'bin/DualSPHysics5.0_NNewtonianCPU_linux64',
+    ],
+    'dirs': ["lib", "bin"],
```

No need to check for `bin` here, as we're already checking for multiple files in that directory.
```suggestion
    'dirs': ["lib"],
```

---

you forgot this one too :)

---

I'm doing to many things in parallel.